site/docs/master/aws-config.md

```diff
@@ -18,7 +18,8 @@ Velero. The tarballs for each release contain the `velero` command-line client.
 of the Velero repository is under active development and is not guaranteed to be stable!_
 
 2. Extract the tarball:
-```bash
+
+```
```

Comment:
I think we need to indent this block since it's under a numbered list -- its indent is off, and the subsequent line gets rendered in a code block as well. (https://deploy-preview-1516--velero.netlify.com/docs/master/aws-config/). Formatting it like https://github.com/heptio/velero/pull/1516/files#diff-039732dd86f30b3b07b42487d3b2cf7eR55 works fine though, and I think is the right way to do it (indent under numbered list items).